docs/dev/troubleshooting/withdraw-delay.md

```diff
@@ -0,0 +1,12 @@
+# Withdraw delay
+
+In order to prevent a quick drain of the protocol in case a critical bug has been discovered and exploited, we are introducing a block execution delay. Each L2 block committed to L1 will have a time lock before it is executed and finalized. This means that there is enough time to verify the effects of the transactions included into a block before the block becomes final. The zkSync team will be monitoring each block and be able to investigate any anomaly (e.g. rapid outflow, unusually large withdrawals, etc).
```

Comment:
It ought to be "Withdrawal delay" ? @githubdoramon

Reply:
indeed.... my bad on this one